packages/api/src/webviewWindow.ts

```diff
@@ -206,27 +197,20 @@ class WebviewWindow {
   /**
    * Set the window and webview background color.
    *
-   * #### Platform-specific:
-   *
-   * - **Android / iOS:** Unsupported for the window layer.
-   * - **macOS / iOS**: Not implemented for the webview layer.
-   * - **Windows**:
-   *   - alpha channel is ignored for the window layer.
-   *   - On Windows 7, alpha channel is ignored for the webview layer.
-   *   - On Windows 8 and newer, if alpha channel is not `0`, it will be ignored.
-   *
-   * @returns A promise indicating the success or failure of the operation.
-   *
    * @since 2.1.0
    */
   async setBackgroundColor(color: Color): Promise<void> {
-    return invoke('plugin:window|set_background_color', { color }).then(() => {
-      return invoke('plugin:webview|set_webview_background_color', { color })
-    })
+    try {
+      await invoke('plugin:window|set_background_color', { color })
+      await invoke('plugin:webview|set_webview_background_color', { color })
+    } catch (err) {
+      const error = new Error('Failed to set background color')
+      ;(error as any).cause = err
+      throw error
```

Comment:
why are you setting the `cause` field like this, it should be 
```suggestion
      throw new Error('Failed to set background color', { cause: err })
```